draft-ietf-cbor-7049bis.md

```diff
@@ -715,14 +721,15 @@ information is in the initial bytes.
 |          25 | IEEE 754 Half-Precision Float (16 bits follow)            |
 |          26 | IEEE 754 Single-Precision Float (32 bits follow)          |
 |          27 | IEEE 754 Double-Precision Float (64 bits follow)          |
-|       28-30 | (Unassigned)                                              |
+|       28-30 | Unassigned, not well-formed now                           |
 |          31 | "break" stop code for indefinite-length items ({{break}}) |
 {: #fpnoconttbl title='Values for Additional Information in Major Type 7'}
 
 As with all other major types, the 5-bit value 24 signifies a
 single-byte extension: it is followed by an additional byte to
 represent the simple value. (To minimize confusion, only the values 32
-to 255 are used.)  This maintains the structure of the initial bytes:
+-to 255 are used.)  This maintains the structure of the
```

Comment:
Is a change intended here>

Reply:
No, I suspect I failed to undo this line break when I merged some other change.